core/src/main/scala/org/apache/spark/deploy/Utils.scala

```diff
@@ -58,6 +59,9 @@ private[deploy] object Utils extends Logging {
       logType: String,
       offsetOption: Option[Long],
       byteLength: Int): (String, Long, Long, Long) = {
+    if (!SUPPORTED_LOG_TYPES.contains(logType)) {
```

Comment:
Do we need to check it in `renderLog` too?

Reply:
Oh, nvm, `renderLog` calls `getLog`.

Reply:
Yes, it does.